docs/monitoring.md

```diff
@@ -269,17 +310,23 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Details for the storage status of a given RDD</td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/logs</code></td>
-    <td>Download the event logs for all attempts of the given application as a zip file</td>
+    <td><code>/applications/[base-app-id]/logs</code></td>
+    <td>Download the event logs for all attempts of the given application as a zip file.
+    </td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/[attempt-id]/logs</code></td>
-    <td>Download the event logs for the specified attempt of the given application as a zip file</td>
+    <td><code>/applications/[base-app-id]/[attempt-id]/logs</code></td>
+    <td>Download the event logs for a specific application as a zip file</td>
```

Comment:
I think the idea is that in all other cases `[app-id]` *must* include the attempt-id when applicable as well.  However `/logs` is special, in that even if there are multiple attempts, there is still an end-point with just base-app-id for getting logs from all attempts.